spec/controllers/account/users_controller_spec.rb

```diff
@@ -189,4 +189,30 @@
       expect(response).to redirect_to root_path
     end
   end
+
+  describe 'Super_admin collection and resourse' do
```

Comment:
What is this name means? You are testing 2 actions (index and show) in one test. Move this tests upper into show and index tests with proper context names please.